website/cue/reference/components/sinks/elasticsearch.cue

```diff
@@ -74,6 +74,19 @@ components: sinks: elasticsearch: {
 	}
 
 	configuration: {
+		api_version: {
+			common:      false
+			description: "The API version of Elasticsearch."
+			required:    false
+			type: string: {
+				default: "auto"
+				enum: {
+					auto: "Auto-detect the api version. Will fail if version endpoint (`/_cluster/state/version`) isn't reachable."
+					v7:   "Use the Elasticsearch 7.x API."
```

Comment:
```suggestion
					auto: "Auto-detect the api version. Will fail if version endpoint (`/_cluster/state/version`) isn't reachable."
					v6:   "Use the Elasticsearch 6.x API."
					v7:   "Use the Elasticsearch 7.x API."
```
Sorry for last minute drive by, but docs are missing the v6?

Reply:
:thinking: 

Good catch . Wondering though if we are intentionally not advertising it per this comment from @jszwedko :

> Great! I'd be fine with just advertising v7 and v8 given previous versions are EOL. If people ask for support for <=v6 we can consider adding them though. So far Vector users generally seem to be on v7 and v8.

Reply:
We don't test against v6 so I'd hesitate to document it as supported. We could add an integration test for v6 and add this though.

Reply:
Ah, ok - I found it odd we added support at all in that case but, like I said bit of a drive by 😄

Reply:
Need to support it per this comment:

> As a note, while v7 and v8 will be advertised as supported, v6 still needs to be supported for sematext sink to function.

Reply:
Oh gotcha - squash and merge then!